backend/app/environments/router.py

```diff
@@ -6,32 +6,35 @@
 
 from app.database.database import get_db_session
 from app.dependencies import only_for_admin
-from app.environments.schemas import (
-    Environment,
-    EnvPlatformConfig,
-    EnvPlatformServiceConfig,
-    GetEnvironment,
+from app.environment_platform_configurations.schema import (
+    EnvironmentPlatformConfiguration,
 )
+from app.environment_platform_service_configurations.schema import (
+    EnvironmentPlatformServiceConfiguration,
+)
+from app.environments.schema import Environment, EnvironmentCreate
 from app.environments.service import EnvironmentService
 
 router = APIRouter(prefix="/envs", tags=["environments"])
 
 
 @router.get("")
-def get_environments(db: Session = Depends(get_db_session)) -> Sequence[GetEnvironment]:
+def get_environments(db: Session = Depends(get_db_session)) -> Sequence[Environment]:
     return EnvironmentService(db).get_environments()
 
 
 @router.post("", dependencies=[Depends(only_for_admin)])
-def create_environment(environment: Environment, db: Session = Depends(get_db_session)):
+def create_environment(
```

Comment:
I just realized this endpoint is redundant since we plan to have one endpoint that handles all the settings. You will not be able to create just an environment.